app/build.gradle

```diff
@@ -30,7 +30,9 @@ license {
 }
 
 dependencies {
-  compile fileTree(dir: 'libs', include: ['*.jar'])
-  compile 'com.android.support:appcompat-v7:22.2.1'
-  compile 'com.feedhenry:fh-android-sdk:2.4.0-SNAPSHOT'
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.android.support:appcompat-v7:23.0.1'
+    compile 'com.android.support:recyclerview-v7:23.0.+'
+    compile 'com.feedhenry:fh-android-sdk:3.0.0-SNAPSHOT'
```

Comment:
We are going to be fixing a bucket of bugs in the SDK.  No reason to link against the broken one.